jenkins/tippecanoe_create_tiles_Jenkinsfile

```diff
@@ -23,13 +22,12 @@ pipeline {
       }
       steps {
         sh 'tippecanoe -Z4 -z8 --no-tiny-polygon-reduction --detect-shared-borders --simplify-only-low-zooms --simplification=5 --force --output-to-directory tile_dir_simple5 hrus.geojson'
-        sh 'tile-join --output-to-directory tile_dir_simple5_data -c hru_data.csv tile_dir_simple5 --force' 
       }
     }
     stage('push to S3') {
       agent any
       steps { 
-        sh 'aws s3 sync tile_dir_simple5_data s3://prod-owi-resources/resources/Application/wbeep/${TIER}/hru_shape/tile_dir_simple5_data'
+        sh 'aws s3 sync tile_dir_simple5 s3://prod-owi-resources/resources/Application/wbeep/${TIER}/hru_shape/tile_dir_simple5'
```

Comment:
figured it makes sense to leave these "non-data" tiles/naked tiles in this bucket? then grab them during the join and then push out to the specified tier?

Reply:
agree